docs/_cli_docs/cli.md

```diff
@@ -442,23 +442,33 @@ This command is the same as running `zapier build` and `zapier upload` in sequen
 
 ## register
 
-> Register a new integration in your account.
+> Register a new integration in your account, or update the existing one if a `.zapierapprc` file is found.
 
 **Usage**: `zapier register [TITLE]`
 
-After running this, you can run `zapier push` to build and upload your integration for use in the Zapier editor.
+This command creates a new integration and links it in the `./.zapierapprc` file. If `.zapierapprc` already exists, it will ask you if you want to update the currently-linked integration, as opposed to creating a new one.
```

Comment:
> This command creates a new integration

Technically, doesn't `zapier init` or `zapier push` create a new integration (depending on how you see it)? `register` more so links together the integration code to a Zapier identifier
WDYT?

Reply:
The current behaviors (as in v13.0.0):

- `zapier register` creates a new integration and adds a link to it if `.zapierapprc` doesn't exist.
- `zapier init` only creates a directory locally. It does not create a new integration on Zapier.
- `zapier push` creates a new version or updates an existing version of an integration.

So the new help text looks good to me!